### PR TITLE
website/integrations: gitlab: update certificate key pair location and specify sha

### LINKED ISSUE #9924: website/integrations: gitlab: update certificate key pair location and specify sha
The documentation tells us to go to:
> Certificate list under Identity & Cryptography

The certificates are now at System -> Certificates. 

The documentation should also specify the certificate fingerprint sha to use (oops).

The associated pull request resolves the issues listed above

### PR DESCRIPTION
**Pull request by dominic-r**
## Details

closes #9924 

---

## Checklist

-   [ ] Local tests pass (`ak test authentik/`)
-   [ ] The code has been formatted (`make lint-fix`)

If an API change has been made

-   [ ] The API schema has been updated (`make gen-build`)

If changes to the frontend have been made

-   [ ] The code has been formatted (`make web`)

If applicable

-   [ ] The documentation has been updated
-   [ ] The documentation has been formatted (`make website`)

### CODE REVIEW

File: website/integrations/services/gitlab/index.md [hunk 1]
```diff
@@ -29,7 +29,8 @@ Under _Advanced protocol settings_, set a certificate for _Signing Certificate_.
 ## GitLab Configuration
 
 Paste the following block in your `gitlab.rb` file, after replacing the placeholder values from above. The file is located in `/etc/gitlab`.
-To get the value for `idp_cert_fingerprint`, go to the Certificate list under _Identity & Cryptography_, and expand the selected certificate.
+
+To get the value for `idp_cert_fingerprint`, navigate to the admin interface's _System_ section and select _Certificates_. Then, expand the chosen certificate and copy the SHA1 Certificate Fingerprint.
```
Comment (BeryJu) [inline website/integrations/services/gitlab/index.md:33]:
```suggestion
To get the value for `idp_cert_fingerprint`, navigate to the Admin interface, expand the  _System_ section and select _Certificates_. Then, expand the selected certificate and copy the SHA1 Certificate Fingerprint.
```

File: website/integrations/services/gitlab/index.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ Under _Advanced protocol settings_, set a certificate for _Signing Certificate_.
 ## GitLab Configuration
 
 Paste the following block in your `gitlab.rb` file, after replacing the placeholder values from above. The file is located in `/etc/gitlab`.
-To get the value for `idp_cert_fingerprint`, go to the Certificate list under _Identity & Cryptography_, and expand the selected certificate.
+To get the value for `idp_cert_fingerprint`, navigate to the Admin interface, expand the  _System_ section and select _Certificates_. Then, expand the selected certificate and copy the SHA1 Certificate Fingerprint.
```
[the remark this answers is not in the document]
Comment (tanberry) [reply website/integrations/services/gitlab/index.md:32]: We have moved to use bold text for UI components (field names, labels, etc) and italics for variables/placeholder values. We have a PR for a new Style Guide... I need to get that finished and published.

**Review by tanberry — CHANGES_REQUESTED** [review]
Thanks so much for this correction! Just a nit on the formatting; as soon as that is changed let's get this merged.

**Review by tanberry — APPROVED** [review]
Thanks! Though it looks like the `lint-prettier` build check is failing... @4d62 can you run prettier again and push?

**Pull request merged**